python/packages/ai/teams/streaming/streaming_response.py

```diff
@@ -278,14 +274,23 @@ async def send_activity(self, activity: Activity) -> None:
             channel_data.stream_id = self._stream_id
             activity.channel_data = StreamingChannelData.to_dict(channel_data)
 
-        entity_args = {
-            "stream_id": channel_data.stream_id,
-            "stream_sequence": channel_data.stream_sequence,
-            "stream_type": channel_data.stream_type,
-        }
-        activity.entities = [
-            Entity(type="streaminfo", **entity_args),
-        ]
+        entity = StreamingEntity(
+            stream_id=channel_data.stream_id,
+            stream_sequence=channel_data.stream_sequence,
+            stream_type=channel_data.stream_type
+        )
+        entities: List[Entity] = [entity]
```

Comment:
NIT: this var isn't really needed, is it?

Reply:
It was by default expecting a list of `StreamingEntity` objects but we also add the `AIEntity` a few lines below, so I noted the parent type to support both

Reply:
Yeah, just a NIT. You can also do this:
`entity: Entity = StreamingEntity(...)`